cfgov/cfgov/settings/base.py

```diff
@@ -177,7 +177,8 @@
 # Absolute path to the directory static files should be collected to.
 STATIC_ROOT = os.environ.get('DJANGO_STATIC_ROOT', '/var/www/html/static')
 
-MEDIA_ROOT = os.path.join(PROJECT_ROOT, 'f')
+MEDIA_ROOT = os.environ.get('MEDIA_ROOT', 
+                            os.path.join(PROJECT_ROOT, 'f'))
```

Comment:
> That fallback won't work if the .env file isn't sourced though.

Should the app work at all if `.env` isn't sourced? Seems like sourcing that is a requirement

> The Django settings file isn't exactly throughout the codebase, in this case (I can't speak for other env vars). If I were coming to the project that would be the first place I'd be looking for default values (indeed, that's where I have been), not a hidden file.

Makes sense, as mentioned in chat my concern was just that Python/JS could access the same envvars, so if defaults are set at a layer before that happens for each it would be slightly more robust.